javascripts/govuk/selection-buttons.js

```diff
@@ -1,6 +1,11 @@
+// DEPRECATED
+// This isn’t needed if you’re using GOV.UK Elements 3.0.0 or above
+
 ;(function (global) {
   'use strict'
 
+  console.warn('Deprecation warning: Custom radio buttons and checkboxes (released in GOV.UK Elements 3.0.0) no longer require this JavaScript.')
```

Comment:
Think we need to guard against console not being there since older browsers will blow up

http://caniuse.com/#search=console